rwengine/src/render/MapRenderer.cpp

```diff
@@ -289,3 +289,9 @@ void MapRenderer::drawBlip(const glm::vec2& coord, const glm::mat4& view,
     renderer->setUniform(rectProg.get(), "colour", glm::vec4(0.0f, 0.0f, 0.0f, 1.0f));
     glDrawArrays(GL_LINE_LOOP, 0, 4);
 }
+
+void MapRenderer::setHUDScale(const float scale) {
```

Comment:
`applyHUDScale` or `scaleHUD` should be better. Now it sounds like normal setter.